helm/tenant/values.yaml

```diff
@@ -273,35 +270,18 @@ ingress:
   api:
     enabled: false
     ingressClassName: ""
-    labels: { }
-    annotations: { }
-    tls: [ ]
+    labels: {}
+    annotations: {}
+    tls: []
     host: minio.local
     path: /
     pathType: Prefix
   console:
     enabled: false
     ingressClassName: ""
-    labels: { }
-    annotations: { }
-    tls: [ ]
+    labels: {}
+    annotations: {}
+    tls: []
     host: minio-console.local
     path: /
     pathType: Prefix
-
-## Use an extraResources template section to include additional Kubernetes resources
-## with the Helm deployment.
-## Example: the following creates the config secret together with the tenant:
-#secrets:
-#  existingSecret: custom-env-configuration
-#extraResources:
```

Comment:
Don't delete this.

Reply:
Ill add that again. I just resolved the other parts

Reply:
I think it can have the mention of being `Deprecated` in the values.yaml right?

Reply:
also, the ability of uncommenting becomes useless due to this part of the code:
```yaml
  ## Secret with default environment variable configurations
  configuration:
    name: {{ .secrets.existingSecret | default .secrets.name }}
```

Reply:
Nothing being deprecated here.